master/internal/db/postgres_rbac_intg_test.go

```diff
@@ -402,6 +422,26 @@ func TestPermissionMatch(t *testing.T) {
 		err = DoesPermissionMatchAll(ctx, userIDClusterAdmin,
 			rbacv1.PermissionType_PERMISSION_TYPE_SET_RESOURCE_QUOTAS, workspaceID)
 		require.NoError(t, err)
+
+		err = DoesPermissionMatch(ctx, userIDClusterAdmin, &workspaceID,
```

Comment:
I think these should be `DoesPermissionMatchAll`!

Reply:
fixed!